img_proof/ipa_cloud.py

```diff
@@ -178,6 +179,7 @@ def __init__(
         self.root_disk_size = int(self.ipa_config['root_disk_size'])
         self.beta = self.ipa_config['beta']
         self.exclude = exclude or []
+        self.instance_options = instance_options or []
```

Comment:
The values come from a chain map so I think it's safer to assume instance_options could be None. Since a call of the method with instance_options=None would take precedence over the default value.

Reply:
Thanks for the clarification.

Reply:
It could be more efficient by checking `is None` instead. That would prevent creating a new empty list to replace an empty list.